src/App.js

```diff
@@ -33,9 +34,22 @@ class App extends React.Component {
     return (
       <div className="wrapper">
         <Nav />
-        <Home employees={employees} owners={owners} pets={pets} />
+        {/* <Home employees={employees} owners={owners} pets={pets} />
         <StaffList employees={employees} />
-        <PetsList pets={pets} />
+        <PetsList pets={pets} /> */}
+
+        <Switch>
+          <Route exact path="/">
+            <Home employees={employees} owners={owners} pets={pets} />
+          </Route>
+          <Route path="/staff">
+            <StaffList employees={employees} />
+          </Route>
+          <Route path="/pets">
+            <PetsList pets={pets} />
+            <Redirect to="/pets/cats/" />
```

Comment:
```suggestion
            <Redirect to="/pets/cats" />
```
the trailing `/` at the end of your path is what's keeping the Redirect from working.